Emby.Server.Implementations/Plugins/PluginManager.cs

```diff
@@ -122,6 +122,12 @@ public IEnumerable<Assembly> LoadAssemblies()
                         ChangePluginState(plugin, PluginStatus.Malfunctioned);
                         continue;
                     }
+                    catch (TypeLoadException ex)
```

Comment:
The point is to stop crashing, so in this case you'd want to catch the general Exception